### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Eclipse Azure Device login_Keyboard: Focus order is not proper wile navigating inside the ‘Azure Device Login’ popup

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3316]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)

MAS Violated: MAS2.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application 
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Select 'Device login' and tab till sign in and press enter
5) Verify If focus order inside the ‘Azure Device Login’ popup is proper

**Actual Result:** 
Keyboard focus is going from 'Device Login' link to the cancel button and when user presses left arrow focus comes on to the ‘copy and Open’ button

**Expected Result:** 
Focus order in the popup should be proper. Focus should go from 'Device Login' link to ‘Copy & Open’ button and then to ‘Cancel’ Button

**User Impact:** 
Keyboard users will find difficulty in navigating if keyboard focus is not moving properly in the popup

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_4_Focus order issue in azure device login popup.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451116/Bug_4_Focus.order.issue.in.azure.device.login.popup.pptx)

**Comment by Flanker32** [comment]
This issue fixed in #3490

**Comment by Tanushagottipati** [comment]
Regressed in the below build and the issue is fixed. Hence closing the bug and attached video for reference
**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse   3.17.0.201909020904
[Bug Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3568881/Bug.Fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749741
